runt.toml

```diff
@@ -481,6 +481,13 @@ fud e {} -s verilog.cycle_limit 500 \
          --to dat -q
 """
 
+[[tests]]
+name = "calyx-py AXI wrapper generation"
+paths = ["yxi/axi-calyx/axi-generator.py"]
+cmd = """
+python3 {}
+"""
```

Comment:
Looks good for now! In the future, you can imagine this using a set of YXI inputs as the test files, so we see different AXI-wrapper code generated for each one.

Reply:
I mentioned this in #1928, which is not very fleshed out currently, but I think(?) points to what you're getting at.